src/BreweryScrapper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class BreweryScrapper
+{
+    protected Crawler $sections;
+    private array $casparyData = [];
+
+    public function casparyScrapper(): void
+    {
+        $url = "https://www.caspary.com/en/references.html";
+        $html = file_get_contents($url);
+        $crawler = new Crawler($html);
+
+        $this->walker = $crawler->filterXPath("//*[@id='referenzmenu']");
+        $fetchedCountry = $this->walker->filterXPath("//span[@class='col-sm-12 title']");
+        $item = $this->walker->filterXPath("//span");
+
+        $countries = [];
+        $fetchedRecords = [];
+        $countryName = "";
+
+        for ($pointerPosition = 0; $pointerPosition < $fetchedCountry->count(); $pointerPosition++) {
+            $countries[] = $fetchedCountry->eq($pointerPosition)->text();
+        }
+
+        for ($pointerPosition = 0; $pointerPosition < $item->count(); $pointerPosition++) {
```

Comment:
Lets use foreach. Could you rename `$item` to `items` or something like that?

Reply:
When I make like` foreach ($items as $item) ...`, I cannot use `->text()`

Reply:
Ok, nevermind, keep it as it is.